### PR TITLE
Cross Entropy Loss forward and back passes derivation

### LINKED ISSUE #2: Derivation for Forward pass and back pass of Cross Entropy loss function

### PR DESCRIPTION
**Pull request by azaher1215**
Fixes #2

### CODE REVIEW

**Comment by COD1995** [comment]
This does not incorporate into the website. Please add to the HTML and follow the style of the website.

**Comment by azaher1215** [comment]
@COD1995 it has been updated to use HTML and was added under the paper's section. I didn't really know where to add it though so please let me know if you want me to create a new section or something or if this is fine as is. 

Looking forward to hear any feedback on the content as well.

**Comment by COD1995** [comment]
The styling and section can be added where you see fit, as this is a public project. Just provide us with the rationale. 

Please provide us with a screenshot like in #38.

**Comment by azaher1215** [comment]
image
image
image
Here is what it looks like

**Comment by COD1995** [comment]
Extra credit noted; Please take this screenshot if extra credit is not given. +1;

**Comment by COD1995** [comment]
closed
